### PR TITLE
Python example

### PR DESCRIPTION
**Pull request by pavankumarbn**
@lorensen @ajpmaclean Please review.

### CODE REVIEW

File: src/Python/ImageData/ImageNormalize.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+import vtk
+
+def main():
+    colors = vtk.vtkNamedColors()
+    # Create an image
```
Comment (ajpmaclean) [inline src/Python/ImageData/ImageNormalize.py]: For consistency, please use the same colours as in the Cxx example.
You can define user colours using vtNamedColors e.g call them `left_bkg`, `right_bkg`
``` python
colors.SetColor("left_bkg", [0.6, 0.5, 0.4, 1.0])
colors.SetColor("right_bkg", [0.4, 0.5, 0.6, 1.0])
```

Comment (pavankumarbn) [reply src/Python/ImageData/ImageNormalize.py]: @ajpmaclean Sure! I will change as suggested.

**Comment by pavankumarbn** [comment]
@ajpmaclean I have made changes as suggested. Please review the same.

**Comment by ajpmaclean** [comment]
Thanks

**Pull request merged**